adm/unix/compileunix.py

```diff
@@ -35,26 +32,23 @@ def gcc(exeFile, srcFiles, includes, libs, defs):
       cmd += " -m32" # always compile in 32bit mode
 
   for include in includes:
-    cmd += " -I\"" + include + "\""
+    cmd += " -I" + include
 
   # defines (tuples)
   for d in defs:
     cmd += " -D" + d[0] + "=" + d[1]
 
   cmd += " -DPLAT_BUILD_VERSION=" + '\\"' + env.buildVersion() + '\\"'
 
+  # libs
+  for lib in libs:
+    cmd += " -l" + lib
+
   # src
   for src in srcFiles:
     cmd += " " + src
 
-  # lib paths
-  for lib in libs:
```

Comment:
why not support -l and -L at the same time ? I think that will be more flexible.